include/kllvm/binary/ProofTraceParser.h

```diff
@@ -337,30 +338,14 @@ class proof_trace_parser {
 
   template <typename It>
   sptr<kore_pattern> parse_kore_term(It &ptr, It end, uint64_t &pattern_len) {
-    if (std::distance(ptr, end) < 11U) {
+    if (std::distance(ptr, end) < 9U) {
       return nullptr;
     }
-    if (detail::read<char>(ptr, end) != '\x7F'
-        || detail::read<char>(ptr, end) != 'K'
-        || detail::read<char>(ptr, end) != 'O'
-        || detail::read<char>(ptr, end) != 'R'
-        || detail::read<char>(ptr, end) != 'E') {
-      return nullptr;
-    }
-    auto version = detail::read_version(ptr, end);
-
-    if (!read_uint64(ptr, end, pattern_len)) {
-      return nullptr;
-    }
-
-    if (std::distance(ptr, end) < pattern_len) {
-      return nullptr;
-    }
-    if (pattern_len > 0 && std::distance(ptr, end) > pattern_len) {
-      end = std::next(ptr, pattern_len);
-    }
```

Comment:
Are these verifications really not necessary anymore? Or do they not fit the new version?

Reply:
I suppose we could probably still verify the 4-byte magic number, come to think of it, but the rest are relevant to things that are no longer part of the serialization format. I will make a small change here.